cdk_lib/dpdash-cdk-stack.ts

```diff
@@ -224,6 +224,9 @@ export class DpdashCdkStack extends cdk.Stack {
       portMappings: [{ containerPort: 8000}],
       environment: {
         MONGODB_HOST: mongoNetworkLoadBalancer.loadBalancerDnsName,
+        ADMIN_EMAIL: process.env.ADMIN_EMAIL,
+        EMAIL_SENDER: process.env.EMAIL_SENDER,
+        HOME_URL: `https://staging.${process.env.BASE_DOMAIN}`,
```

Comment:
should we make this whole thing an env var?

Reply:
I thought about that but figured it was repetitive. It's good that it's more explicit, but we'd have to add both `DEV_HOME_URL` and `PROD_HOME_URL` to the Github settings as opposed to using the `BASE_DOMAIN` with standardized subdomains.

Reply:
But maybe that doesn't matter, depending on where we fall on the open infrastructure questions.

Reply:
so you're saying we need to specify the environment in our environment variable names? That feel very un-environment variable 😄 . Is there a way we could have a single `HOME_URL` that gets set per environment that we're deploying to?

Reply:
Sorta? From the _pipeline's_ perspective yes, because the pipeline is aware of both dev/staging and eventually prod. From the _app's_ perspective it's just `HOME_URL`, but in the Github Actions variables we'd need both.

Reply:
gotcha. I was thinking we might not settle on `staging` as the subdomain, but I have no idea. Pulling that thread a little more, should we hard-code some of these values for now and switch them to env vars later on when we know more?

Reply:
I'd just as soon leave them as-is. I think we'll need more flexibility later but I do think all the env vars we're using now will stick around